report.sh

```diff
@@ -13,11 +13,11 @@ red="\033[31;40m"
 none="\033[0m"
 
 clear
-TIME=1
+readonly TIME=1
 echo " "
```

Comment:
TIME is variable not a constant, so it should be "time". 
If it was constant, it should be set as readonly:
`readonly TIME=1`
Also, you may want to allow user to override the default value :

`TIME=${TIME:-'1'}`

Reply:
Okay, this is needed info :)

Reply:
This PR is not meant for you to merge (actually this will break your script, because TIME is NOT a constant). The PR is there to give you a few pointers  ;)

Reply:
Actually, the real fix for this part is:
`time=${time:-'1'}`

Reply:
oh okay, my bad :( I'll change it accordingly.